ports/pcl/portfile.cmake

```
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO PointCloudLibrary/pcl
-    REF d98313133b014553ab1b1b5b112f9aade837d55c # pcl-1.11.1
-    SHA512 4d60f34d4fbf0a4b4caf9cc4391af471ebd260b3bbac106d45e5ff38448894ea4dc82d5320c2e395c537a4414eb13c6a6a0eb6c13e4e1cc1d831d4bf24067966
+    REF 4d59ab8eeee0ed45865562015450d77c730044ea # pcl-1.11.1
```

```suggestion
    REF 4d59ab8eeee0ed45865562015450d77c730044ea # pcl-1.11.1+patch
```
There should be some extra comments.

---

Okay!